dht.go

```diff
@@ -265,23 +265,38 @@ func (d *DHT) AddNode(addr string) {
 // Asks for more peers for a torrent.
 func (d *DHT) getPeers(infoHash InfoHash) {
 	closest := d.routingTable.lookupFiltered(infoHash)
-	for _, r := range closest {
-		// *remoteNode is nil if it got filtered.
-		if r != nil {
-			d.getPeersFrom(r, infoHash)
+	if len(closest) == 0 {
+		for _, s := range strings.Split(d.config.DHTRouters, ",") {
```

Comment:
May be worth creating a helper function for strings.Split(d.config.DHTRouters, ",").